forte/processors/ner_predictor.py

```diff
@@ -348,3 +354,183 @@ def default_configs(cls):
 
         configs.update(more_configs)
         return configs
+
+    
+class BioBERTNERPredictor(FixedSizeBatchProcessor):
```

Comment:
We currently use Python typing for most variables. It would be great if you can add the typing to the variables in this class.

Reply:
Got it. I can add typing to variables where appropriate. 

When I observed the `CoNLLNERPredictor` class, I noticed that only variables that we expect to be passed outside a function gets typing. Would you describe that as the guiding principle to typing in this repo?

Reply:
Actually, I have merged this and we don't have to worry about the typing right now, since I am considering converting this as an example.

`CoNLLNERPredictor` is a bit old so some convention is not fully enforced.

Reply:
I see. Thanks for providing the context